### PR TITLE
Typo in survey-instrument.md

### PR DESCRIPTION
**Pull request by leveled-up**
Typo "communty" instead of "community" in line 857.

### CODE REVIEW

File: survey-instrument.md [hunk 1]
```diff
@@ -842,7 +842,7 @@ Q54
 **PUBLIC-DATA-RELEASE-CONSENT**
 (single choice + textbox)
 
-As a reminder, the responses from this survey will be released as an open data set, available for anyone to analyze and study. Aside from responses, the only other information that will be released is a binary indicator of whether a respondent was sourced from GitHub.com traffic or an off-site open source communty. Response data will **not** be linked to any metadata such as IP address/geography, browser information, survey language, or identity on any project or site.
+As a reminder, the responses from this survey will be released as an open data set, available for anyone to analyze and study. Aside from responses, the only other information that will be released is a binary indicator of whether a respondent was sourced from GitHub.com traffic or an off-site open source communtiy. Response data will **not** be linked to any metadata such as IP address/geography, browser information, survey language, or identity on any project or site.
```
Comment (valscion) [inline survey-instrument.md]: There's a typo in your typo fix. `communtiy` should be `community` :smile:

Comment (anaskhan96) [inline survey-instrument.md]: "There's a typo in your typo fix."
I think this is the first time I came across a sentence like this on GitHub.

Comment (valscion) [inline survey-instrument.md]: Leveling up with the typo fixes with @leveled-up

Comment (leveled-up) [inline survey-instrument.md]: Exactly. @valscion

**Comment by franniez** [comment]
Thanks for the effort 😄 We actually cut this question (see https://github.com/github/open-source-survey/pull/70) for a couple of reasons (it created confusion about what to do with partial responses; we don’t have bandwidth to  custom edit responses by hand, it contained duplicate info from the introduction and just made a long survey even longer, etc.)

**Pull request closed without merging**